### DISCUSSION TITLE
OpenScan mini would not stop

### DISCUSSION THREAD

**Discussion by memento42429** [OpenScanEu/OpenScan2#87] [General]
OpenScan mini has been behaving strangely since yesterday. Specifically, after you start shooting and take the specified number of shots, shooting will start again from the beginning. It won't stop.
I have tried all currently available versions, including Beta. I also tried a clean install, but the same behavior occurred.
Device: Raspberry Pi 3 Model B Rev 1.2
OS: Raspbian GNU/Linux 11 (bullseye) Jan 20 2022 13:58:22
Current version: 2024-09-09

What can I do to solve this problem? Please lend me your wisdom.

**Comment by OpenScanEu** [comment]
could you post the content of the log file (update&info tab) for further diagnostic?
it might be an issue with the limited RAM of the Pi 3B (500mb), but i am not sure about that at this point

**Reply by memento42429** [reply]
Thank you for your reply. 
After leaving it for a while, it appeared to be cured. I'll check the log file again if I run into the problems.
